test/unit/transmitter-payload.test.ts

```diff
@@ -1,13 +1,19 @@
 import * as tap from 'tap';
+import { readFileSync } from 'fs';
+import { join } from 'path';
 
-import kubeScannerTypes = require('../../src/scanner/types');
-import payload = require('../../src/transmitter/payload');
-import transmitterTypes = require('../../src/transmitter/types');
-import podSpecFixture = require('../fixtures/pod-spec.json');
-import config = require('../../src/common/config');
+import { config } from '../../src/common/config';
+import { IScanResult } from '../../src/scanner/types';
+import * as payload from '../../src/transmitter/payload';
+import { ILocalWorkloadLocator, IWorkload } from '../../src/transmitter/types';
+import {V1PodSpec} from "@kubernetes/client-node";
```

Comment:
Any chance this can move to the top with the other node_modules imports? (also the quotes are `"` instead of `'`)

Reply:
yes you right